### DISCUSSION TITLE
Adding support for translating the Help documentation

### DISCUSSION THREAD

**Discussion by legofanlovessayori** [Floorp-Projects/Floorp#597] [Feature Request]
Adding Translations for the **Floorp specific** Help documentation kinda makes sense since we also translated the Browser itself.

Just to add to this aswell, would make sense to redirect the Help Bookmark to the Floorp help instaid of the general help.

**Comment by regularentropy** [comment]
Hi. Thank you for your suggestion!

This is planned, but it will take some time.

**Reply by CutterKnife** [reply]
Just as the Floorp browser is being localized by Crowdin, it would be good if the support documentation were translated by the community.
